### ISSUE TITLE
[Change node-fetch] iconv-loader issue

### ISSUE THREAD

**Issue by subdavis** [perry-mitchell/webdav-client#73]
Due to andris9/encoding#18

Dependency chain:
`webdav -> node-fetch -> encoding `

I won't go over the gritty details, but because of the dependency on encoding, I'm seeing
```
WARNING in ./node_modules/encoding/lib/iconv-loader.js
9:12-34 Critical dependency: the request of a dependency is an expression
 @ ./node_modules/encoding/lib/iconv-loader.js
 @ ./node_modules/encoding/lib/encoding.js
 @ ./node_modules/node-fetch/lib/body.js
 @ ./node_modules/node-fetch/index.js
 @ ./node_modules/webdav/dist/request.js
 @ ./node_modules/webdav/dist/index.js
...
```

While this bug isn't yours to solve, if there is another alternative to node-fetch, that would be awesome to use.  Meanwhile, I'm going to have to use a forked version of webdav with the dependency removed (since I intend to use in-browser)

Thanks!

**Comment by perry-mitchell** [comment]
Hi,

Yes this is a known issue with node-fetch when using it in webpack. You could of course use [node-noop to stop this from happening](https://github.com/buttercup/buttercup-browser-extension/blob/master/webpack.config.js#L84). For the time being we won't be changing node-fetch, as this is primarily a node library (even though I use it extensively in the browser), but I'll definitely consider changing it out for another library in a 2.0 release.

**Comment by perry-mitchell** [comment]
Will most likely be fixed by perry-mitchell/webdav-client#99
